package.json

```diff
@@ -145,28 +145,28 @@
     "@types/redis": "^2.8.32",
     "@types/superagent": "^4.1.15",
     "@types/supertest": "^2.0.11",
-    "@typescript-eslint/eslint-plugin": "^5.37.0",
-    "@typescript-eslint/parser": "^5.37.0",
+    "@typescript-eslint/eslint-plugin": "^5.38.1",
+    "@typescript-eslint/parser": "^5.38.1",
     "audit-ci": "^6.3.0",
     "axe-core": "^4.4.3",
     "cheerio": "^1.0.0-rc.12",
     "concurrently": "^7.3.0",
-    "cypress": "^8.6.0",
-    "cypress-axe": "^0.13.0",
+    "cypress": "^10.8.0",
```

Comment:
@dhirajshettymoj you can't just go from 9 to 10, there are large changes required in the config.